integration/heapster_api_test.go

```diff
@@ -190,6 +207,31 @@ func getSchema(fm kubeFramework, svc *kube_api.Service) (*api_v1.TimeseriesSchem
 	return &timeseriesSchema, nil
 }
 
+func getModelMetrics(fm kubeFramework, svc *kube_api.Service, pod *kube_api.Pod) (*api_v1.MetricResultList, error) {
+	url := fmt.Sprintf("/api/v1/model/namespaces/%s/pod-list/%s/metrics/%s",
```

Comment:
Well, I hope so. I can still poke all of the endpoints and check if they don't crash (but probably in the next PR).